pkg/config/setup/config.go

```diff
@@ -925,6 +925,7 @@ func InitConfig(config pkgconfigmodel.Setup) {
 	config.BindEnvAndSetDefault("inventories_min_interval", 0) // 0 == default interval from inventories
 	// Seconds to wait to sent metadata payload to the backend after startup
 	config.BindEnvAndSetDefault("inventories_first_run_delay", 60)
+	config.BindEnvAndSetDefault("inventories_network_use_hostname_resolver", false) // resolve the hostname to get the IP address
```

Comment:
We're trying to deprecate the `inventory` prefix:

```suggestion
	config.BindEnvAndSetDefault("metadata_ip_resolution_from_hostname", false) // resolve the hostname to get the IP address
```

Reply:
Done in [address review](https://github.com/DataDog/datadog-agent/pull/35145/commits/9445dd6ad2f9317c46b0b5f134d517ca9ccb19a4)